src/js.cookie.js

```diff
@@ -140,10 +140,10 @@
 		api.get = function (key) {
 			return api.call(api, key);
 		};
-		api.getJSON = function () {
-			return api.apply({
+		api.getJSON = function (key) {
+			return api.call({
```

Comment:
Any byte size impact or none?

Reply:
I haven't had coffee, but I think it's an improvement even:

<img width="836" alt="bildschirmfoto 2018-03-21 um 07 41 04" src="https://user-images.githubusercontent.com/21918/37697126-4dca4c22-2cdb-11e8-9943-1f6ee9d1ff35.png">

Reply:
o.O wow